core/utils.py

```diff
@@ -88,18 +88,31 @@ def read_manual(fua: int, proj_crs: str | int | pyproj.CRS) -> geopandas.GeoData
     )
 
 
+def read_osmnx(
+    fua: int | str, proj_crs: str | int | pyproj.CRS
+) -> geopandas.GeoDataFrame:
+    """Read OSM roads from parquet format; return bare columns."""
+    return geopandas.read_parquet(_fua_path(fua, "osmnx")).to_crs(proj_crs)
+
+
 def read_parenx(
-    fua: int, option: str, proj_crs: str | int | pyproj.CRS
+    fua: int, proj_crs: str | int | pyproj.CRS, option: str
 ) -> geopandas.GeoDataFrame:
     """Read in prepared parenx data."""
 
     return (
-        geopandas.read_parquet(_fua_path(fua, "parenx", option=option))
+        geopandas.read_parquet(_fua_path(fua, f"parenx-{option}"))
         .explode(ingore_index=True, index_parts=False)
```

Comment:
So do we need this `explode` for `parenex` read-ins?

Reply:
@jGaboardi ok just discussed with @martinfleis , consensus is to add `.explode()` in all read in funcs, will do that now